site/src/modules/resources/AgentVersion.tsx

```diff
@@ -24,7 +24,7 @@ export const AgentVersion: FC<AgentVersionProps> = ({
   );
 
   if (status === agentVersionStatus.Updated) {
-    return <span>Updated</span>;
+    return null;
```

Comment:
Just checking if we want to do this since this looks to be outside of the scope.

Reply:
sneaking in out of scope things is my favorite 😝 but yes, this was intentional